asb-ballerina/messageSender.bal

```diff
@@ -22,21 +22,26 @@ import ballerina/jballerina.java as java;
 public isolated client class MessageSender {
 
     final string connectionString;
-    final string entityPath;
     final handle senderHandle;
+    final string queueName;
+    final string topicName;
 
     # Initializes the connector. During initialization you can pass the [Shared Access Signature (SAS) authentication credentials](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas)
     # Create an [Azure account](https://docs.microsoft.com/en-us/learn/modules/create-an-azure-account/) and 
     # obtain tokens following [this guide](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-quickstart-portal#get-the-connection-string). 
     # Configure the connection string to have the [required permission](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-sas).
     # 
     # + connectionString - Connection String of Azure service bus
-    # + entityPath - Name or path of the entity (e.g : Queue name, Topic name)
+    # + queueName - Queue name
+    # + topicName - Topic name
     public isolated function init(@display {label: "Connection String"} string connectionString, 
```

Comment:
Lets define the constructor as 

init (string entityName, EntityType entityType) {..}

EntityType contains --> QUEUE, TOPIC